cernopendata/modules/fixtures/data/articles/virtual-machines-cms-2011/cms.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "title": "CMS 2010 Virtual Machines: How to install",
+    "title": "CMS 2011 Virtual Machines: How to install",
```

Comment:
Please rename also the files `*-cms-2011` so that there won't be any confusion when we add the 2010 ones 😉

Reply:
Oh, I see it's renamed everywhere, sorry 😉